src/xrpld/app/main/Application.cpp

```diff
@@ -45,9 +45,8 @@
 #include <xrpld/app/misc/ValidatorKeys.h>
 #include <xrpld/app/misc/ValidatorSite.h>
 #include <xrpld/app/paths/PathRequests.h>
+#include <xrpld/app/rdb/RelationalDatabase.h>
```

Comment:
Is it still needed?

Reply:
Yes, for `std::unique_ptr<RelationalDatabase> mRelationalDatabase;` on line 218.